thaomonster/ACNH-Collections
# Refactor/first look through

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds stubbing to testing and takes a look through to make sure commented out code and console.logs have been deleted.

### Is this a feature or Fix/Refactor?  
Refactore

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

Stubbing would not work until I changed the beforeEach to just a before.
https://docs.cypress.io/guides/guides/test-retries.html#Introduction

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`cypress/fixtures/profilePageData.json` lines: 1-49;
`cypress/integration/profilePage-test.js ` lines 2-4;
`cypress/integration/villagerList-test.js` lines 3-5;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` branch
2. run `npm run cypress`
3. watch the green checkmarks! 

### Steps To Take
>Any questions and possible steps moving forward.

None.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
